### DISCUSSION TITLE
error: Move abort in 0x1::object: ENO_UNGATED_TRANSFERS(0x50003): The object does not have ungated transfers enabled #353Question Title

### DISCUSSION THREAD

**Discussion by tharunhack** [aptos-labs/aptos-developer-discussions#354] [Questions] [answered]
### Describe your question in detail.

while transferring the nfts using ts-sdk 

### What error, if any, are you getting?

error: Move abort in 0x1::object: ENO_UNGATED_TRANSFERS(0x50003): The object does not have ungated transfers enabled #353Question Title

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by gregnazario** [comment, marked as answer]
This code means the NFT trying to be transferred is soulbound and can't be transferred.  https://aptos.dev/en/build/smart-contracts/digital-asset#soulbound-tokens  https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/doc/object.md#0x1_object_ENO_UNGATED_TRANSFERS
